molecule/default/molecule.yml

```diff
@@ -0,0 +1,29 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+platforms:
+  - name: instance-1
+    image: centos:8
+    command: "/usr/sbin/init"
+    volumes:
+    - "/sys/fs/cgroup:/sys/fs/cgroup:rw"
+    privileged: True
+    groups:
+      - ipfs_cluster
+provisioner:
+  name: ansible
+  inventory:
+    group_vars:
+      ipfs_cluster:
+        ipfs_cluster_secret: "81643eba8fedcc4aa31fef6459e8b852a97c3cf3d0489158cb85166e8255e912"
+    host_vars:
+      instance-1:
+        ipfs_peer_id: "QmUdZCogpVVrPMxdPHhtT2KuUuxNVi1uLb7UC4HwHBqejb"
+        ipfs_private_key: "CAASqAkwggSkAgEAAoIBAQC3NFhK6XMF9tDDky2aoCdwnNsIzk0DnvkVtfTsM1/fCqnrlMDzvVOY7tsMFBNBHsjCAuLwjUMxtLTgY2+8b8vil5TTuYk0A2nCCTwyxqKHoxwn5RW3J5HlBBMgTV9vKtRyrDONFnRwvVmUbCaxblOp74tl03KEfF3zH46+vz8bzVeNdqDrGmEIc1tWnERddFAeqcM5r8fhNVsVInCWFThqFA2qUahif43Uo1B/gBySnyXWr61htiHoWBXqPK8eFk4FCBodhs+Ct81KE4QQhV9OQaZeQkChGezIPidvwiojY8Lr40xBsRn0H95AKYWwdo0lTbAJ7dtFuBinZrTNdSz/AgMBAAECggEAM7Ism+7mCy/LVvSn8LLyjh9k20ZDixsH1G2E0EXq7qg2rIhMLjslZMk5vF3J1R1Xrw9bdF+YL2V7iLnmZLjzQlxdnaUEJ04zQJKUjwtn/3gyULue9hDPZV1R/FJ49IiUbEtVPKGyd8GmjnA6NuUX7KLwpeoVrXbel+U2xXJ332MGU1uydteq4iH27FYKYK+6UZUzHLd+V9BOwzGXEXG22Z25hT8n0mp0u1LcJy5tecAj3PAPW0uhsvAFX95c/jkQRkUNj8mntTb2AL2aw4MGNj8UbLYg0EdBRfE3V+esyj0ob7OUt0Nk5d6yCOBXDSZdD/P1mDYOgiL6JpaG9445YQKBgQDvgSKClI4TmXPerRNHhN8ZKbgLy1Hcu9hMs/pmv4Up0KQ6VbrsArXD1KMglrpsXZojgKeDuLmzLnVmMeslZyYqlWS3wWXeJZ2Ms2PlcnNDQlxlwJbJapNO4V9UZTIGcydg5lJ0QWL7HVGANNvM0Jg4wFQbvjj+tXS6kVYNz4JCDwKBgQDD0owaNJNIfo9JvkXFCNafhepiSJSqxXozo1zqVLCjuesmzfgplo+6qjiJJCMbK4rZbb8DCHK1NQmROahyWmyKDNl4sOMHnH6ckO7HUILLx9M4LKzJj1B7lQX7hEjy3XZAXwT+xx4t/lyF8iG3Ta4Kxcd8ya7tQW8YBMB3LUaWEQKBgQCNBe+JDenedoO4gRaSDRXEXkFFiJGKNH03JN5zM6A4L+cThjlLEIlwZrsqzDW26yKAdxerwtBtRhTQeOIpouPhPRgR7umJdYKgP1y51LszxlZbf5wJuxxD3QZhzjg3fgSY1OWGGJ4smGXN5NHrdoO7HtwnfclXiQbTeBJAjYf4VwKBgGK8AlAkOi7PQUgZW+xg7HY6e1da6TrBnz/yGqM8WGP1Fg/Wwk2V0Jp5wU/EnL7SUYwXvbu1wsCDIXPL8pbV6OwiVnR4G8B2s5AVs49jlM9SzbkBLY3jaN0m4oFSzcVGzJzgTBB6LUlbeyM+VqmpzRs5ZcARoNWXrIAbxT9R4MlBAoGBAJPGfycrr284aF67Q30drAfxXE5AFw11DBqE6KfRdfDe02PNnlClI0BDbvA1BJyg2kzAeJvOZAAvuiw/eNuwOnqhtnoBjzJLW2EnEUu7++m/joa6nq4LZ5eHwoR+exwA6l0GH6pItHuDPESQq8JqrByLfEwsS5Jj4OxILBD7cBO8"
```

Comment:
ed25519 keys are much shorter now, could use that to make things prettier.

Reply:
fixed in https://github.com/hsanjuan/ansible-ipfs-cluster/pull/10/commits/f7bca19e00109894cbb15a02f78f4ebdab5aeb03